src/libtriton/bindings/python/objects/pyTritonContext.cpp

```diff
@@ -647,18 +603,7 @@ namespace triton {
 
                 /* Check the call */
                 if (ret == nullptr) {
-                  PyObject* type      = nullptr;
-                  PyObject* value     = nullptr;
-                  PyObject* traceback = nullptr;
-
-                  /* Fetch the last exception */
-                  PyErr_Fetch(&type, &value, &traceback);
-
-                  std::string str = PyStr_AsString(PyObject_Str(value));
-                  Py_XDECREF(type);
-                  Py_XDECREF(value);
-                  Py_XDECREF(traceback);
-                  throw triton::exceptions::Callbacks(str);
+                  throw triton::exceptions::PyCallbacks();
```

Comment:
Can you explain this?

Reply:
If an exception happen in a python callback it is better to have the traceback than a single line error. The easiest way is to not format a new python error for this specific kind of exception and return null in the native python functions. I thus added specific catches for try clauses that do use callbacks directly or indirectly.

Reply:
Got it.